cmd/signal/grpc/main.go

```diff
@@ -97,8 +101,14 @@ func parse() bool {
 	flag.StringVar(&addr, "a", ":50051", "address to use")
 	flag.StringVar(&metricsAddr, "m", ":8100", "merics to use")
 	flag.IntVar(&verbosityLevel, "v", -1, "verbosity level, higher value - more logs")
+	flag.StringVar(&paddr, "paddr", "", "pprof listening address")
 	help := flag.Bool("h", false, "help info")
 	flag.Parse()
+
+	if paddr == "" {
+		paddr = getEnv("paddr")
```

Comment:
Maybe we can use "PPROF_ADDR":)

Reply:
I used it to be in sync with all-rpc version of the server:
https://github.com/pion/ion-sfu/blob/191ae903e730d8e36169d02faf2b33fb416c69e9/cmd/signal/allrpc/main.go#L107